backend/lambdas/api/authorizer/tests/test_get_user.py

```diff
@@ -0,0 +1,148 @@
+import json
+import os
+import unittest
+
+from authorizer.handlers import _get_update_or_create_user
+from unittest.mock import patch
+
+EMAIL_DOMAIN_ALIASES = [
+    {
+        "new_domain": "company1.com",
+        "old_domains": ["company1.com"],
+        "email_transformation": {"new_email_regex": "[.]", "old_email_expr": "_"},
+    },
+    {
+        "new_domain": "company2.com",
+        "old_domains": ["company2.com"],
+        "email_transformation": {"new_email_regex": "([a-z]+)\.([a-z]+)", "old_email_expr": "\\2.\\1"},
+    },
+    {"new_domain": "newcompany.com", "old_domains": ["company.com"]},
+]
```

Comment:
Would it make sense to add a test case that both changes the domain name and transforms the email?

Reply:
I went ahead and added a new test case